bot/slack.py

```diff
@@ -203,20 +206,17 @@ def perform_bot_cmd(msg, private=True):
 
 def perform_text_replacements(text: str) -> Union[str, None]:
     """Replace first matching word in text with a little easter egg"""
-    words = text.lower().split()
-    strip_chars = "?!"
-    matching_words = [
```

Comment:
this was redundant, nice concise solution @JnyJny

Reply:
do you remember what happened in Slack? would be nice to document it in a test, thanks!